docssrc/src/arguments.md

```diff
@@ -199,26 +197,23 @@ We first register the first `/kill` command as normal:
 
 </div>
 
-Now we declare our command with arguments for our second command. Then, we can register our second command `/kill <target>` as usual:
+This gives us the ability to run both `/kill` and `/kill <target>` with the same command name "kill", but have different results based on the arguments used.
 
-<div class="multi-pre">
+In this example, we use the simpler, inline `.withOptionalArguments(Argument... arguments)` method to register our argument. There is no difference to using this method as opposed to explicitly declaring a list and using `.withOptionalArguments(List<Argument> arguments)`, so feel free to use whichever method you want!
```

Comment:
I think this is good to mention, but it doesn't make sense in the current document, because we've not told the user that there is a `.withOptionalArguments(List<Argument> arguments)` method yet.

Typically, when I write my documentation, I like to declare the API first (if I'm not leading with an example first), then reference the API later. For example, in [Argument suggestions](https://commandapi.jorel.dev/8.7.0/argumentsuggestions.html), we show the reader the list of methods (1), then we reference them in the text (2):
<img width="395" alt="image" src="https://user-images.githubusercontent.com/4613171/210541384-336f32ae-f81a-4d38-b99f-a07c63eb1d8e.png">